lib/base/utility.cpp

```diff
@@ -1301,6 +1301,12 @@ tm Utility::LocalTime(time_t ts)
 #endif /* _MSC_VER */
 }
 
+time_t Utility::MkTime(tm *timeptr)
+{
+	timeptr->tm_isdst = -1;
+	return mktime(timeptr);
+}
```

Comment:
Given that the wrapper is also named `MkTime()`, this somewhat looks like that this should be the behavior of `mktime()` and what it actually does is to be considered a bug. So for example, on days where the clock jumps backwards, the actual value of `tm_isdst` might be important for something like 2:30 and probably each call to `mktime()` should be assessed individually if `tm_isdst` has to be reset or not.

In general, how much testing went into this PR?

Reply:
1. That’s exactly what I wanted to archive! 🎉
2. Enough testing to make me flipping the table. But feel free to test even more. As said: this is a fragile area.